File: Selenium/Project/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.firefox import GeckoDriverManager

service = webdriver.FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:
    driver.get("https://alchemy.hguy.co/lms")

    firstInfoBoxTitle = driver.find_element(By.XPATH, "(//h3)[1]").text

    try:
        assert firstInfoBoxTitle == "Actionable Training"
        print("The title of the first Info Box is Actionable Training")
    except:
        print("The title of the first Info Box does not match")

    driver.quit()

File: Selenium/Project/Activity3.java
package projects;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {

    public static void main(String[] args){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/lms");

        String firstInfoBoxTitle = driver.findElement(By.xpath("(//h3)[1]")).getText();

        if(firstInfoBoxTitle.equals("Actionable Training")){
            System.out.println("The title of the first Info Box is Actionable Training");
            driver.quit();
        }else{
            System.out.println("The title of the first Info Box does not match");
        }

    }
}
